compiler/rustc_middle/src/ty/layout.rs

```diff
@@ -2483,10 +2483,9 @@ impl<'tcx> ty::Instance<'tcx> {
                 // `src/test/ui/polymorphization/normalized_sig_types.rs`), and codegen not keeping
                 // track of a polymorphization `ParamEnv` to allow normalizing later.
```

Comment:
Should this comment be removed?

Reply:
Nope, this just reverts the change here from #86993. I had made this change before because `normalize_erasing_regions` couldn't handle the escaping vars (so it was pushed to only occur with `polymorphize` enabled). But other changes required a different approach in `query/normalize.rs` to allow them anyways. So I reverted this back. *But* theoretically this could stay the same.